src/less/components/file/index.less

```diff
@@ -1,3 +1,4 @@
+@import 'mixins';
```

Comment:
Was it intentional to add this to the LESS file? 
Were you intending to add it to the _SCSS_ file?

Reply:
@BY00565233, you should not have to touch this file for this PR #653.  

Please revert changes.